<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageExtraWidget</class>
 <widget class="QWidget" name="ManageExtraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>614</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="extraList">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ParentId</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Setup</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UpTime</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Info</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>State</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="showMoreButton">
       <property name="text">
        <string>Show More</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showAllButton">
       <property name="text">
        <string>Show All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Submit Extra</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>extraList</tabstop>
  <tabstop>showMoreButton</tabstop>
  <tabstop>showAllButton</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>removeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
